src/Pyz/Zed/AntelopeDataImport/Business/Step/AntelopeWriterStep.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Zed\AntelopeDataImport\Business\Step;
+
+use Orm\Zed\Antelope\Persistence\PyzAntelopeQuery;
+use Spryker\Zed\DataImport\Business\Model\DataImportStep\DataImportStepInterface;
+use Spryker\Zed\DataImport\Business\Model\DataSet\DataSetInterface;
+
+class AntelopeWriterStep implements DataImportStepInterface
+{
+    public const KEY_NAME = 'name';
```

Comment:
One good practise is to do the column names in a DataSet file - but I do not know where to put this, i want to avoid the model folder for this. Also putting it to Shared namespace seems to be an overhead

```php
<?php
 
namespace Spryker\Zed\{YourModule}DataImport\Business\Model\DataSet;
 
interface {YourModule}DataSetInterface
{
    public const COLUMN_IN_YOUR_CSV_IMPORT_FILE = 'column name of import file';
}
```